app/gitcoinbot/gitcoinbotActions.py

```diff
@@ -0,0 +1,193 @@
+'''
```

Comment:
Can you use `"""` when possible? Typically `'''` can cause issues for docstrings.  `"""` is PEP-257/258

Also, can you rename this file to conform to snake_case? You could probably go with `app/gitcoinbot/actions.py` since it's already in the `gitcoinbot` app.